src/components/Organisms/AssetPageFooter/AssetPageFooter.interface.ts

```diff
@@ -0,0 +1,8 @@
+import * as H from 'history'
+
+export interface AssetPageFooterProps {
+  links?: { url: string; icon: string; text: string }[]
```

Comment:
`interface Link { url: string; icon: string; text: string }`

```suggestion
  links?: Link[]
```

Reply:
@matthill8286 are you saying that I should extend the `Link` interface?

Reply:
@matthill8286 this has been updated